lib/data/weather/service/weather_service_impl.dart

```diff
@@ -1,36 +0,0 @@
-import 'dart:io';
-
-import 'package:wheather_app/data/api_url_provider.dart';
-import 'package:wheather_app/data/weather/data_source/weather_api_data_source.dart';
-import 'package:wheather_app/data/weather/mapper/weather_dto_to_weather_mapper.dart';
-import 'package:wheather_app/domain/weather/model/weather.dart';
-import 'package:wheather_app/domain/weather/service/weather_service.dart';
-import 'package:injectable/injectable.dart';
-
-@LazySingleton(as: WeatherService)
-class WeatherServiceImpl implements WeatherService {
```

Comment:
Maybe Repository is better name for that? Repository means that youre getting data, service sometimes can have more responsibilities